testing/functional/tests/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -0,0 +1,85 @@
+export default class DashboardInstitutionObject {
+  loginWithBCEID() {
+    return cy.contains("Login with BCeID");
+  }
+
+  loginInWithBCEIDtext() {
+    return cy.contains("Log in with");
+  }
+
+  bceidInputText() {
+    return cy.get("#user");
+  }
+
+  passwordInputText() {
+    return cy.get("#password");
+  }
+
+  continueButton() {
+    return cy.get('input[type="submit"]');
+  }
+
+  dashboardButton() {
+    return cy.contains("Dashboard");
+  }
+
+  notificationButton() {
+    return cy.contains("Notifications");
+  }
+
+  programsButton() {
+    return cy.contains("Programs");
+  }
+
+  activeApplicationsButton() {
+    return cy.contains("Active Applications");
+  }
+
+  programInfoRequestsButton() {
+    return cy.contains("Program Info Requests");
+  }
+
+  confirmationOfEnrollment() {
+    return cy.contains("Confirmation of Enrollment");
+  }
+
+  homeButton() {
+    return cy.contains("Home");
+  }
+
+  manageInstitutionButton() {
+    return cy.contains("Manage Institution");
+  }
+
+  profileButton() {
+    return cy.contains("PROFILE");
+  }
+
+  profileIconButton() {
+    return cy.xpath("//button[contains(@class,'v-btn--size-default mr-5')]");
```

Comment:
I believe we are checking the class of the user profile rounded button here. 
v-btn--size-default is the class supplied by vuetify framework. For all the buttons, it uses different types of css class and hence if the size of the button or type of the button changes then class v-btn--size-default may not be part of the button. IMO I recommend using the class **_v-btn_** here which is present in all vuetify button components.
![image](https://user-images.githubusercontent.com/54600590/163084040-f564efc7-b8d1-4d50-ba7f-e24550699ceb.png)

Correct me If I am missing anything.

Reply:
I have concerns about how we are searching for the elements on selectors in general. As much as possible we should try to follow the practices on the below link, IMO.
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Also, I have noticed selecting elements by id. Mainly if we select form.io elements by id, it will append an alphanumeric bunch of characters  to the id. for e.g ek9fo68 in here. 

`institutionOperatingNameInputText() {
    return cy.get("#ek9fo68-operatingName");
  }`

This alphanumeric can be very dynamic. I would like to sync up with @bhavikccits to get my understanding clear.